astroid/__pkginfo__.py

```diff
@@ -16,7 +16,7 @@
 
 modname = 'astroid'
 
-version = '1.6.6'
+version = '1.6.7'
```

Comment:
Can you remove the version bump? We usually do it separately than backports.

Reply:
Reverted